cypress/e2e/specialFeatures/atiAnalytics/assertions/index.js

```diff
@@ -4,6 +4,13 @@ import {
   getATIParamsFromURL,
   interceptATIAnalyticsBeacons,
 } from '../helpers';
+import {
+  VIEW_EVENT,
+  VIEWABILITY_CLICK_EVENT,
+} from '../../../../../src/app/lib/analyticsUtils/analytics.const';
+
+const usesReverbViewabilityModel = ({ useReverb }) =>
```

Comment:
This probably doesn't need to use an object param now.
```suggestion
const usesReverbViewabilityModel = useReverb =>
```